submissions/rubyAdeniregun/DOM/product.js

```diff
@@ -24,5 +24,79 @@ let submitForm1 = document.getElementById('verifyForm'),
         footeremailvalidation(e);
         return true;  
     }
-    
-    submitForm1.addEventListener("submit", verifyForm);
\ No newline at end of file
+
+submitForm1.addEventListener("submit", verifyForm);
+
+
+//EXAMPLE
+
+let item = document.getElementById("quantityBox")
+    button = document.getElementById("buyNow");
+    shippingCost = document.getElementById("shippingCost");
+    cost = document.getElementById("totalCost");
+    price = document.getElementById("price");
+
+//initial value
+let priceValue = price.innerHTML,
+    costValue = cost.innerHTML;
+//RegEx
+const Regex = /^[1-9]/;
+function calculateQuantity() {
+    if (!item.value.match(Regex)) {
+        button.disable = true
+        button.style = "background-color: grey"
+        item.nextElementSibling.innerHTML = "⚠️Please enter a valid quantity";
+    } else {
+        button.disabled = false;
+        item.nextElementSibling.innerHTML = "";
+    }
+}
+function calculatePrice() {
+    if (item.value >= 1) {
+        let perPrice = (2500 * item.value)
+        price.innerHTML = perPrice
+    } else if (item.value === "") {
```

Comment:
Are you saying that if I don't choose any quantity or if is it empty, you still want to assign a price and cost value to it? Well, I don't think it makes sense though if you ask me. I think a better user experience would have been that if it is empty, you should display an error message saying "Please pick or enter a quantity of the item you want to but`